apps/storefront/src/pages/CompanyOrderList/index.test.tsx

```diff
@@ -18,19 +18,20 @@ import {
 } from 'tests/test-utils';
 import { when } from 'vitest-when';
 
+import { CompanyStatus, CustomerRole, UserTypes } from '@/types';
+
 import {
   CompanyOrderNode,
   CompanyOrderStatuses,
-  CustomerOrderStatus,
   GetCompanyOrders,
-} from '@/shared/service/b2b/graphql/orders';
-import { CompanyStatus, CustomerRole, UserTypes } from '@/types';
+  OrderStatus,
+} from '../order/orders';
```

Comment:
Import these from the smaller version contained within the orders page instead of the old-full global version.